packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -575,13 +575,11 @@ export class TreeCheckout implements ITreeCheckoutFork {
 				cursor,
 			);
 			const tree = jsonableTreeFromCursor(cursor);
-			if (tree !== undefined) {
```

Comment:
What is the significance of this change?

Reply:
I'm just cleaning up. `jsonableTreeFromCursor` cannot return undefined.